docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -2325,82 +2328,206 @@ following abbreviations:
-        , TIMESTAMP '2006-09-12 11:59:58.999999'
-        )
+SELECT DATEDIFF( SECOND
+                 , TIMESTAMP '2006-09-12 11:59:58.999998'
+                 , TIMESTAMP '2006-09-12 11:59:58.999999'
+               )
+FROM DUAL;
 ```
 
 * This function returns the value 1 because a one-second boundary is
 crossed even though the two timestamps differ by only one microsecond.
 +
 ```
-DATEDIFF( SECOND
-        , TIMESTAMP '2006-09-12 11:59:58.999999'
-        , TIMESTAMP '2006-09-12 11:59:59.000000'
-        )
+SELECT DATEDIFF( SECOND
+                 , TIMESTAMP '2006-09-12 11:59:58.999999'
+                 , TIMESTAMP '2006-09-12 11:59:59.000000'
+               )
+FROM DUAL;
 ```
 
-* This function returns the value of 0.
+[[date_difference_in_minute]]
+==== Date Difference in MINUTE
+
+* This function returns the value of 2 because a two-minute boundary is crossed.
 +
 ```
-DATEDIFF( YEAR
-        , TIMESTAMP '2006-12-31 23:59:59.999998'
-        , TIMESTAMP '2006-12-31 23:59:59.999999'
-        )
+SELECT DATEDIFF( MINUTE
+                 , TIMESTAMP '2011-12-30 08:54:55'
+                 , TIMESTAMP '2011-12-30 08:56:01'
+               ) 
+FROM DUAL;
+```
+
+[[date_difference_in_day]]
+==== Date Difference in DAY
+
+* This function returns the value of -29.
++
+```
+SELECT DATEDIFF( DAY
+                 , DATE '2008-03-01'
+                 , DATE '2008-02-01'
+               ) 
+FROM DUAL;
+```
+
+* This statement calculates how long do buyers have to wait. 
++
+```
+SELECT id, DATEDIFF( DAY, order_date, delivery_date ), price 
+FROM orders 
+ORDER BY price DESC;
 ```
 
-* This function returns the value of 1 because a year boundary is
-crossed.
 +
 ```
-DATEDIFF( YEAR
-        , TIMESTAMP '2006-12-31 23:59:59.999999'
-        , TIMESTAMP '2007-01-01 00:00:00.000000'
-        )
+ID           (EXPR)       PRICE 
+-----------  -----------  ------
+
+     700300          145     926
+     312203          147     621
+     800660           23     568
+     100350          160     543
+     500450          148     324
+     700510          141     229
+     100210            3     228
+     600480          151     197
+     300380          154     183
+     200490          227     123
+     200320          153      91
+     400410          158      65
+     600250          143      32
+
+--- 13 row(s) selected.
 ```
 
-* This function returns the value of 2 because two WEEK boundaries are
+[[date_difference_in_week]]
+==== Date Difference in WEEK
+
+* This function returns the value of 1 because only a one-week boundary is
 crossed.
 +
 ```
-DATEDIFF(WEEK, DATE '2006-01-01', DATE '2006-01-09')
+SELECT DATEDIFF( WEEK
+                 , DATE '2006-01-01'
+                 , DATE '2006-01-09'
+               )
+FROM DUAL;
 ```
 
-* This function returns the value of -29.
+[[date_difference_in_quarter]]
+==== Date Difference in QUARTER
+
+* This function returns the value of 3 because a three-quarter boundary is crossed.
```

Comment:
Possible wordsmith: "This function returns the value of 3 because three quarter boundaries are crossed."

Reply:
Thanks Dave, your comments have been incorporated. :)